.gitignore

```diff
@@ -51,3 +51,4 @@ log/
 risedev-components.user.env
 riselab-components.user.env
 risedev-compose.yml
+payload.txt
```

Comment:
Should be in grafana/.gitignore? Otherwise all payload.txt will be ignored. I think it's common to have a file of such name inside a repo, but we just want to ignore the one in grafana directory.